core/server/analytics.js

```diff
@@ -161,4 +161,30 @@ Meteor.methods({
       zone_name: traits.zoneName,
     });
   },
+  analyticsReaction(traits) {
```

Comment:
Maybe it should be done in another PR to keep consistency over the whole file but `properties` is more correct than `traits` when using `track`